Database/Corrections/cataObjectFixes.lua

```diff
@@ -328,6 +328,11 @@ function CataObjectFixes.Load()
         [460014] = {
             [objectKeys.name] = "Mark of the World Tree",
         },
+        [460015] = {
+            [objectKeys.name] = "Durnholde Keep Barrel",
+            [objectKeys.spawns] = {[zoneIDs.OLD_HILLSBRAD_FOOTHILLS]={{76.60,68.80},{77.14,66.30},{74.68,68.79},{69.09,62.61},{68.03,59.82}}},
+            [objectKeys.zoneID] = zoneIDs.OLD_HILLSBRAD_FOOTHILLS,
+        },
```

Comment:
[This wowhead comment](https://www.wowhead.com/cata/quest=29598/tarethas-diversion#comments:id=1629784) suggests, the objects are just named "Barrel". While there are a huge amount of Barrels around the world, it's good to add these special barrels, with the correct locations.

It is important though, that the name of the objects match the ingame name. The reason is, that there is no way to get any ID of an object in the game (e.g. when hovering over it). So in order to show tooltips on quest related objects, Questie uses the name to matches it to objects.

Reply:
They aren't objects in game though - they don't seem to be anything and have no mouse over/tooltip (in the same way that a plant doesn't). Plus, the barrels aren't actually the objective per se, it's 0/1 buildings set on fire. The more I think about it, I reckon this should be for an extraObjective rather than objective. Would that make a difference to the tooltip issue?

Reply:
I see. Well if they are not any real objects and don't have any tooltip then I'd also say using `extraObjectives` would be better.

Reply:
https://github.com/Questie/Questie/pull/6444/commits/07e01acc7c6c5396eb9346456bbc125ce7437dd0 and https://github.com/Questie/Questie/pull/6444/commits/0545cf3899906464c231b1312ac8a05feb64842f